docs/languages/jsconfig.md

```diff
@@ -43,7 +43,7 @@ By default the JavaScript language service will analyze and provide IntelliSense
 
 ### Using the `"exclude"` property
 
-The `exclude` attribute (a glob pattern) tells the language service what files are and are not part of your source code. This keeps performance at a high level. If IntelliSense is slow, add folders to your `exclude` list (VS Code will prompt you to do this if it detects the slow down).
+The `exclude` attribute (a glob pattern) tells the language service what files are or are not part of your source code. This keeps performance at a high level. If IntelliSense is slow, add folders to your `exclude` list (VS Code will prompt you to do this if it detects the slow down).
```

Comment:
I think we might just want to say "what files are not part of your source code". The `exclude` filter doesn't list any files that are in the project.